op-node/sources/eth_client.go

```diff
@@ -182,6 +216,21 @@ func (s *EthClient) InfoAndTxsByLabel(ctx context.Context, label eth.BlockLabel)
 	return s.blockCall(ctx, "eth_getBlockByNumber", string(label))
 }
 
```

Comment:
Longer term these should definitely be consolidated to work with a type similar to upstream's `rpc.BlockNumberOrHash`

Reply:
Agreed, `ENG-2673` covers this